src/infrastructure/controllers/store/root-store/navigation-store.tsx

```diff
@@ -0,0 +1,32 @@
+import { action, makeAutoObservable, observable } from 'mobx';
+import { PagesEnum } from '~/domain/interfaces/enum/pages-enum';
+
+class NavigationStore {
+    navigate: any = () => {};
```

Comment:
Ici pour le any on devrait pouvoir le remplacer par 
```suggestion
    navigate: (route: string) => void = (route: string) => void;
```